examples/basics/src/components/Welcome.astro

```diff
@@ -0,0 +1,210 @@
+---
+import astroLogo from '../assets/astro.svg';
+import background from '../assets/background.svg';
+---
+
+<div id="container">
+	<img id="background" src={background.src} alt="" fetchpriority="high" />
+	<main>
+		<section id="hero">
+			<a href="https://astro.build"
+				><img src={astroLogo.src} width="115" height="48" alt="Astro Homepage" /></a
+			>
+			<h1>
+				To get started, open the <code><pre>src/pages</pre></code> directory in your project.
+			</h1>
+			<section id="links">
+				<a class="button" href="https://docs.astro.build">Read Our Docs</a>
```

Comment:
In a narrow viewport this button gets stretched in a weird way:

![Starter template in a narrow viewport. The “Read Our Docs” button is stretched across the viewport width but the text inside is aligned to the left leaving a strangely unbalanced space on the right](https://github.com/user-attachments/assets/eccd5569-7a10-4394-bcce-80ad5939a788)

What I’d probably do here is avoid the `@media` query on links and use `flex-wrap: wrap` so they wrap naturally once there’s not enough horizontal space for them.

(The alternative I guess is `text-align: center` on the buttons, but that might look a bit odd with the Discord button given it doesn’t have a clear outline?)

Reply:
I did that on purpose to make the button easy to click!

Reply:
Tried something else, let me know if that works for you